webui/src/App.js

```diff
@@ -651,43 +662,47 @@ function FSRWebUI(props) {
   return (
     <div className="App">
       <Router>
-        <Navbar bg="light">
-          <Navbar.Brand as={Link} to="/">FSR WebUI</Navbar.Brand>
-          <Nav>
-            <Nav.Item>
-              <Nav.Link as={Link} to="/plot">Plot</Nav.Link>
-            </Nav.Item>
-          </Nav>
-          <Nav className="ml-auto">
-            <NavDropdown alignRight title="Profile" id="collasible-nav-dropdown">
-              {profiles.map(function(profile) {
-                if (profile === activeProfile) {
-                  return(
-                    <NavDropdown.Item key={profile} style={{paddingLeft: "0.5rem"}}
-                        onClick={ChangeProfile} active>
-                      <Button variant="light" onClick={RemoveProfile}>X</Button>{' '}{profile}
-                    </NavDropdown.Item>
-                  );
-                } else {
-                  return(
-                    <NavDropdown.Item key={profile} style={{paddingLeft: "0.5rem"}}
-                        onClick={ChangeProfile}>
-                      <Button variant="light" onClick={RemoveProfile}>X</Button>{' '}{profile}
-                    </NavDropdown.Item>
-                  );
-                }
-              })}
-              <NavDropdown.Divider />
-              <Form inline onSubmit={(e) => e.preventDefault()}>
-                <Form.Control
-                    onKeyDown={AddProfile}
-                    style={{marginLeft: "0.5rem", marginRight: "0.5rem"}}
-                    type="text"
-                    placeholder="New Profile" />
-              </Form>
-            </NavDropdown>
-          </Nav>
-        </Navbar>
+        <>
+          <Navbar bg="light">
+            <Navbar.Brand as={Link} to="/">FSR WebUI</Navbar.Brand>
+            <Nav>
+              <Nav.Item>
+                <Nav.Link as={Link} to="/plot">Plot</Nav.Link>
+              </Nav.Item>
+            </Nav>
+            <Button onClick={PersistThresholds}>Persist thresholds</Button>
```

Comment:
If possible, I think the button should live to the left of the profile picker on the right side of the navbar, since I think the left side of the bar is more for specific "pages" of the ui.